test/service.test.ts

```diff
@@ -196,3 +196,14 @@ test('can set publishNotReadyAddresses', () => {
   const spec = Testing.synth(chart)[0].spec;
   expect(spec.publishNotReadyAddresses).toBeTruthy();
 });
+
+test('can be imported and used by an ingress', () => {
+  const chart = Testing.chart();
+
+  const service = kplus.Service.fromServiceName(chart, 'Service', 'service-name');
+
+  // Now we must specify a port for imported services
+  service.exposeViaIngress('/hello', { port: 8080 });
+  const ingress = Testing.synth(chart)[0];
+  expect(ingress).toMatchSnapshot();
+});
```

Comment:
We should change the implementation to allow for the following API:

```suggestion
test('can be imported and used by an ingress', () => {
  const chart = Testing.chart();

  // import an existing service that exposes port 8000 and is called 'service-name'. 
  const service = kplus.Service.fromServiceName(chart, 'Service', 'service-name', { port: 8000 });

  // same api as with a managed service
  service.exposeViaIngress('/hello');
  const ingress = Testing.synth(chart)[0];
  expect(ingress).toMatchSnapshot();
});
```